core/src/test/scala/dev/atedeg/ecscala/dsl/ECScalaDSLTest.scala

```diff
@@ -40,4 +44,114 @@ class ECScalaDSLTest extends AnyWordSpec with Matchers with ECScalaDSL {
       world.getComponents[Position] shouldBe empty
     }
   }
+
+  "remove (Component) from world" should {
+    "work the same way as the entity.removeComponent() method" in new WorldFixture with ComponentsFixture {
+      val position = Position(1, 2)
+      val velocity = Velocity(3, 4)
+      val entity1 = world hasAn entity
+
+      entity1 withComponents {
+        position &: velocity
+      }
+      remove {
+        position &: CNil
+      } from entity1
+
+      world.getComponents[Velocity] should contain(Map(entity1 -> velocity))
+    }
+  }
+
+  "remove { Component1() &: Component2() from world" should {
+    "work the same way as multiple entity.removeComponent() method calls" in new WorldFixture with ComponentsFixture {
+      val position = Position(1, 2)
+      val velocity = Velocity(3, 4)
+      val entity1 = world hasAn entity
+
+      entity1 withComponents {
+        position &: velocity
+      }
+      remove {
+        position &: velocity
+      } from entity1
+
+      world.getComponents[Position] shouldBe empty
+      world.getComponents[Velocity] shouldBe empty
+    }
+  }
+
+  "world - myEntity" should {
+    "work the same way as the world.removeEntity() method" in new WorldFixture {
+      val entity1 = world hasAn entity
+      world - entity1
+      world.entitiesCount shouldBe 0
+    }
+  }
+
+  "remove (entity1, entity2) from world" should {
+    "work the same way ad the world.removeEntity() method" in new WorldFixture {
+      val entity1 = world hasAn entity
+      val entity2 = world hasAn entity
+
+      remove {
+        entity1
+      } from world
```

Comment:
Could be written in one line 
```suggestion
      remove { entity1 } from world
```

Reply:
scalafmt must have gone crazy